buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/measurement/coverage/GenerateMeasurementsTask.groovy

```diff
@@ -84,12 +87,23 @@ public class GenerateMeasurementsTask extends DefaultTask {
                 'firebase-functions': 6,
                 'firebase-inappmessaging-display': 7,
                 'firebase-storage': 8,
-                'firebase-datatransport': 9
+                'firebase-datatransport': 9,
+
+                // 'firebase-common-ktx' and 'firebase-firestore-ktx' has been
+                // made a subproject of their corresponding java projects in PR
+                // #409 and #426. Old mappings are kept for backward
+                // compatibility reason.
+                'firebase-common:ktx': 10,
+                'firebase-firestore:ktx': 11,
+
+                // 'firebase-storage:test-app' was added in PR #303.
+                'firebase-storage:test-app': 12
```

Comment:
This is a test app that is only used for instrumentation tests. I don't think we need to measure coverage for it.

Reply:
Done. Removed.